src/domains/migration/components/SelectPolygonAddress/SelectPolygonAddress.tsx

```diff
@@ -14,6 +14,7 @@ const AddressAvatar = ({ address }: any) => {
 			/>
 		);
 	}
+
 	return <Avatar address={address} size="sm" noShadow />;
```

Comment:
@alfonsobries we could use the EthereumAvatar instead of a regular one. We are using it for the other steps too
```suggestion
	return <EthereumAvatar address={address} size="sm" noShadow />;
```